app/views/users/show.html.erb

```diff
@@ -14,13 +14,33 @@
       </div>
     </div>
     <% @user.top_posts.each_with_index do |post, index| %>
-        <a href="<%= user_post_path(@user, post) %>" class="h-full flex items-center border-gray-200 border p-4 rounded-lg mb-4">
+        <div class="h-full flex items-center border-gray-200 border p-4 rounded-lg mb-4">
           <div class="flex-grow"> 
-            <h2 class="text-gray-900 title-font font-medium" >Post #<%= index + 1 %></h2>
-            <p class="text-gray-500"><%= post.text %></p>
-            <p class="text-gray-500">Comments: <%= post.comments.count %> Likes: <%= post.likes.count %> </p>
+            <a href="<%= user_post_path(@user, post) %>">
+              <h2 class="text-gray-900 title-font font-medium" >Post #<%= index + 1 %></h2>
```

Comment:
- [ ] Good job adding forms, however, let's display our post title to make sure it has been created and for a good user interface 😃, kindly replace the line:

` <h2 class="text-gray-900 title-font font-medium" >Post #<%= index + 1 %></h2>` 
### With
`<h2 class="text-gray-900 title-font font-medium" ><%=post.title %></h2>`